content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -38,8 +38,16 @@ See the Google Kubernetes Engine (GKE) guide to [configuring cluster access for
 
     ```
     export PIPELINE_VERSION={{% pipelines/latest-version %}}
-    kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+    kubectl apply -f https://storage.googleapis.com/ml-pipeline/pipeline-lite/$PIPELINE_VERSION/crd.yaml
+    kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
+    kubectl apply -f https://storage.googleapis.com/ml-pipeline/pipeline-lite/$PIPELINE_VERSION/namespaced-install.yaml
     ```
+   **Note**: the above approach is introduced since KFP version 0.2.0. For an older
```

Comment:
Suggested revision:

**Note**: The above approach is introduced in Kubeflow Pipelines version 0.2.0. For older versions please run the following instead:

Reply:
Done.